Resources/ServerInfo/_NF/Guidebook/Rules/9_Pirates.xml

```diff
@@ -3,11 +3,19 @@
 
 Pirates must follow their code.
 
+    - [color=#ff0000]The safe zone rules still apply to pirates.[/color]
     - Pirate crews may never exceed a total force of four members, including the Captain, sentient pets and cyborgs.
     -- Round start pets (e.g. Clarrrpy) do not count towards this limit.
     - There may only be one Pirate Captain and one First Mate at any time.
+    - Pirates may not operate without a First Mate or Captain present.
+    -- If the leader of the pirates has been captured or killed, the remaining pirates may make efforts to rescue them.
```

Comment:
They're welcome to party at the cove, hand themselves in and get some prisoner RP going, or just cryo with their tails between their legs. Most attempt breakouts and end up getting arrested, anyway. 
No, they can't open new slots if the old FM and/or cap are still alive, because they're still pirates and that would be in breach of the 4-pirate limit rule. 
Frankly I think pirates are more trouble than they're worth and should be removed and replaced with a better antag, but that's not the topic of conversation here.